<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>loginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
                stop:0 #e3f2fd, stop:0.5 #f0f4f8, stop:1 #ffffff);  /* ËóçÁôΩÊº∏Â±§ËÉåÊôØ */
}

/* Â¢ûÂä†Ê®ôÈ°åÊ®£Âºè */
QMainWindow {
    border: 3px solid #1976d2;
    border-radius: 15px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font: 28pt &quot;Microsoft YaHei UI&quot;;
    font-weight: bold;
    color: #0d47a1;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
                stop:0 rgba(255,255,255,0.9), stop:1 rgba(255,255,255,0.7));
    border: 3px solid #1976d2;
    border-radius: 20px;
    padding: 10px;
    text-align: center;
}
</string>
    </property>
    <property name="text">
     <string>üè¶ ATM Banking System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font: 12pt &quot;Microsoft YaHei UI&quot;;
    color: #1565c0;
    background-color: rgba(255,255,255,0.8);
    border: 2px solid #1976d2;
    border-radius: 8px;
    padding: 4px;
}
</string>
    </property>
    <property name="text">
     <string>üïê 2025Âπ¥6Êúà8Êó• ÊòüÊúüÊó•</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="AccountEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font: 14pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #1976d2;
    border-radius: 12px;
    padding: 8px 12px;
    background-color: #ffffff;
    color: #1565c0;
    selection-background-color: #bbdefb;
}

QLineEdit:focus {
    border: 3px solid #0d47a1;
    background-color: #f3f9ff;
}

QLineEdit:hover {
    border: 3px solid #1e88e5;
    background-color: #fafbfc;
}
</string>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Password">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 15pt &quot;Microsoft YaHei UI&quot;;
    color: #0d47a1;               /* Ê∑±ËóçËâ≤ÊñáÂ≠ó */
    background-color: transparent;
    font-weight: bold;
}
</string>
       </property>
       <property name="text">
        <string> PASSWORD : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PasswordEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font: 14pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #1976d2;
    border-radius: 12px;
    padding: 8px 12px;
    background-color: #ffffff;
    color: #1565c0;
    selection-background-color: #bbdefb;
}

QLineEdit:focus {
    border: 3px solid #0d47a1;
    background-color: #f3f9ff;
}

QLineEdit:hover {
    border: 3px solid #1e88e5;
    background-color: #fafbfc;
}
</string>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="LoginButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font: 16pt &quot;Microsoft YaHei UI&quot;;
    font-weight: bold;
    border: 3px solid #1976d2;
    border-radius: 15px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                stop:0 #64b5f6, stop:1 #1976d2);  /* ËóçËâ≤Êº∏Â±§ */
    color: #ffffff;             /* ÁôΩËâ≤ÊñáÂ≠ó */
    padding: 12px 24px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                stop:0 #81c784, stop:1 #388e3c);  /* Á∂†Ëâ≤Êº∏Â±§ */
    border: 3px solid #388e3c;
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                stop:0 #2e7d32, stop:1 #1b5e20);  /* Ê∑±Á∂†Ëâ≤Êº∏Â±§ */
    border: 3px solid #1b5e20;
    padding: 14px 22px;
}
</string>
       </property>
       <property name="text">
        <string>üîê Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Account">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: 15pt &quot;Microsoft YaHei UI&quot;;
    color: #0d47a1;               /* Ê∑±ËóçËâ≤ÊñáÂ≠ó */
    background-color: transparent;
    font-weight: bold;
}
</string>
       </property>
       <property name="text">
        <string>   ACCOUNT : </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="adminloginButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>540</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font: 52pt &quot;Microsoft YaHei UI&quot;;
    color: #ff6f00;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
                stop:0 #fff3e0, stop:1 #ffcc02);  /* Ê©ôËâ≤Êº∏Â±§ */    border: 3px solid #ff8f00;
    border-radius: 15px;
    padding: 8px 16px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
                stop:0 #c8e6c9, stop:1 #4caf50);  /* Á∂†Ëâ≤Êº∏Â±§ */
    border-color: #388e3c;  /* Á∂†Ëâ≤ÈÇäÊ°Ü */
    color: #1b5e20;         /* Ê∑±Á∂†Ëâ≤ÊñáÂ≠ó */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
                stop:0 #a5d6a7, stop:1 #2e7d32);  /* Ê∑±Á∂†Ëâ≤Êº∏Â±§ */
    border-color: #1b5e20;
    color: #ffffff;
    padding: 10px 14px;
}
</string>
    </property>
    <property name="text">
     <string>üîß </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #ff6f00;
    font: 12pt &quot;Microsoft YaHei UI&quot;;
    font-weight: bold;
    background-color: rgba(255, 255, 255, 0.8);
    border-radius: 4px;
    padding: 2px;
}
</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
